doc/protocols/scion-header.rst

```diff
@@ -473,3 +473,237 @@ Next Header
+            \times \text{q} 
+    \end{align}
+
+TsRel has a precision of :math:`\text{21 \mu s}` and covers at least 
+one day (1 day and 63 minutes). When sending packets at high speeds 
+(more than one packet every :math:`\text{21 \mu s}`) or when using 
+multiple cores, collisions may occur in TsRel. To solve this 
+problem, the source further identifies the packet using PckId.
+
+PckId
+  A 4-byte identifier that allows to distinguish two packets with 
+  the same TsRel. Every source is free to set PckId arbitrarily, but 
+  we recommend to use the following structure:
+
+::
+
+     0                   1                   2                   3
+     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |    CoreID     |                  CoreCounter                  |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+        
+CoreID
+  Unique identifier representing one of the cores of the source host. 
+
+CoreCounter
+  Current value of the core counter belonging to the core specified 
+  by CoreID. Every time a core sends an EPIC packet, it increases 
+  its core counter (modular addition by 1).
+
+Note that the Packet Timestamp is at the very beginning of the 
+header, this allows other components (like the replay suppression 
+system) to access it without having to go through any parsing 
+overhead. 
+
+.. To achieve an even higher precision of the timestamp, the 
+.. source is free to allocate additional bits from the PckId to TsRel 
+.. for this purpose.
+
+Hop Validation Fields (PHVF and LHVF)
+-------------------------------------
+::
+
+     0                   1                   2                   3
+     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                             PHVF                              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |                             LHVF                              |
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+Those 4-byte fields contain the Hop Validation Fields of the 
+penultimate and the last hop of the last segment. 
+
+EPIC Header Length Calculation
+------------------------------
+The length of the EPIC Path header is the same as the SCION Path
+header plus 8 bytes (Packet Timestamp), and plus 8 bytes for the 
+PHVF and LHVF.
+
+Procedures
+----------
+**Control plane:**
+The beaconing process is the same as for SCION, but the ASes not 
+only add the 6 bytes of the truncated MAC to the beacon, but further 
+append the remaining 10 bytes.
+
+**Data plane:**
+The source fetches the path, including all the 6-byte short hop 
+authenticators and the remaining 10 bytes of the authenticators, 
+from the path server. We will refer to the fully assembled 16-byte 
```

Comment:
The source should probably get the full authenticators from a _hidden_ path server.

Reply:
Added that now.